test/rubocop/cop/minitest/assert_with_two_arguments_test.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class AssertWithTwoArgumentsTest < Minitest::Test
+  def test_registers_offense_when_second_argument_is_not_a_string
+    assert_offense(<<~RUBY)
+      class FooTest < Minitest::Test
+        def test_do_something
+          my_list = [1, 2, 3]
+          assert(3, my_list.length)
+          ^^^^^^^^^^^^^^^^^^^^^^^^^ Did you mean to use `assert_equal(3, my_list.length)`?
+        end
+      end
+    RUBY
+  end
```

Comment:
Can you add the testcase?

```ruby
  def test_registers_offense_when_second_argument_is_a_variable
    assert_offense(<<~RUBY)
      class FooTest < Minitest::Test
        def test_do_something
          my_list = [1, 2, 3].length
          assert(3, my_list_length)
          ^^^^^^^^^^^^^^^^^^^^^^^^^ Did you mean to use `assert_equal(3, my_list_length)`?
        end
      end
    RUBY
  end
```

Reply:
Added in d8e98bfec03f0732b4d3d15eb0bc206c4ff32d68